mathlibtools/lib.py

```diff
@@ -583,9 +586,8 @@ def build(self) -> None:
     def clean_mathlib(self, force: bool = False) -> None:
         """Restore git sanity in mathlib"""
         if self.is_mathlib:
-            if not self.is_dirty or force:
-                assert self.repo
-                self.repo.head.reset(working_tree=True)
+            # the mathlib folder contains user edited files - do not touch it!
+            pass
```

Comment:
Why doing a `pass` rather than `return`? Isn't it only obfuscating things? And the comment is also very misleading, nothing tells you the folder contains user edited file here.

Reply:
No preference between `pass` and `return` here, either sounds good to me. How about this as the comment?
```
# if this is mathlib itself then the lean files are part of the working tree - do not touch them!
```